### ISSUE TITLE
Circuit Z side doesn't show up as 'connected_endpoint'

### ISSUE THREAD

**Issue by funzoneq** [netbox-community/netbox#4622]
### Environment
* Python version: 3.7.3
* NetBox version: v2.7.12

### Steps to Reproduce
1. Create a new circuit with type l2circuit
2. Connect A side to Device1:ge-0/0/2
3. Connect Z side to Device2:ge-0/0/0

### Expected Behavior
I expect to see a cable with a connected device for both A & Z side.
![circuit](https://user-images.githubusercontent.com/242967/81577241-7cc0fd00-93a9-11ea-95ce-035c083448ce.png)

### Observed Behavior

Only A side shows a connected_endpoint:

```
curl -X GET "https://netbox.example.com/api/circuits/circuit-terminations/?circuit_id=85"
{
  "count": 2,
  "next": null,
  "previous": null,
  "results": [
    {
      "id": 133,
      "circuit": {
        "id": 85,
        "url": "https://netbox.example.com/api/circuits/circuits/85/",
        "cid": "1"
      },
      "term_side": "A",
      "site": {
        "id": 47,
        "url": "https://netbox.example.com/api/dcim/sites/47/",
        "name": "EVE-ASVA01",
        "slug": "eve-asva01"
      },
      "port_speed": 1000000,
      "upstream_speed": null,
      "xconnect_id": "",
      "pp_info": "",
      "description": "",
      "connected_endpoint_type": "dcim.interface",
      "connected_endpoint": {
        "id": 790,
        "url": "https://netbox.example.com/api/dcim/interfaces/790/",
        "device": {
          "id": 33,
          "url": "https://netbox.example.com/api/dcim/devices/33/",
          "name": "imr01-eve-asva01.tingfiber.net",
          "display_name": "imr01-eve-asva01.tingfiber.net"
        },
        "name": "ge-0/0/2",
        "cable": 174,
        "connection_status": {
          "value": true,
          "label": "Connected"
        }
      },
      "connection_status": {
        "value": true,
        "label": "Connected"
      },
      "cable": {
        "id": 174,
        "url": "https://netbox.example.com/api/dcim/cables/174/",
        "label": ""
      }
    },
    {
      "id": 134,
      "circuit": {
        "id": 85,
        "url": "https://netbox.example.com/api/circuits/circuits/85/",
        "cid": "1"
      },
      "term_side": "Z",
      "site": {
        "id": 48,
        "url": "https://netbox.example.com/api/dcim/sites/48/",
        "name": "EVE-ASVA02",
        "slug": "eve-asva02"
      },
      "port_speed": 1000000,
      "upstream_speed": null,
      "xconnect_id": "",
      "pp_info": "",
      "description": "",
      "connected_endpoint_type": null,
      "connected_endpoint": null,
      "connection_status": null,
      "cable": {
        "id": 179,
        "url": "https://netbox.example.com/api/dcim/cables/179/",
        "label": ""
      }
    }
  ]
}

curl -X GET "https://netbox.example.com/api/dcim/cables/?id=174"
{
  "count": 1,
  "next": null,
  "previous": null,
  "results": [
    {
      "id": 174,
      "termination_a_type": "circuits.circuittermination",
      "termination_a_id": 133,
      "termination_a": {
        "id": 133,
        "url": "https://netbox.example.com/api/circuits/circuit-terminations/133/",
        "circuit": {
          "id": 85,
          "url": "https://netbox.example.com/api/circuits/circuits/85/",
          "cid": "1"
        },
        "term_side": "A"
      },
      "termination_b_type": "dcim.interface",
      "termination_b_id": 790,
      "termination_b": {
        "id": 790,
        "url": "https://netbox.example.com/api/dcim/interfaces/790/",
        "device": {
          "id": 33,
          "url": "https://netbox.example.com/api/dcim/devices/33/",
          "name": "imr01-eve-asva01.tingfiber.net",
          "display_name": "imr01-eve-asva01.tingfiber.net"
        },
        "name": "ge-0/0/2",
        "cable": 174,
        "connection_status": {
          "value": true,
          "label": "Connected"
        }
      },
      "type": "",
      "status": {
        "value": "connected",
        "label": "Connected",
        "id": true
      },
      "label": "",
      "color": "",
      "length": null,
      "length_unit": null
    }
  ]
}

curl -X GET "https://netbox.example.com/api/dcim/cables/?id=179"
{
  "count": 1,
  "next": null,
  "previous": null,
  "results": [
    {
      "id": 179,
      "termination_a_type": "circuits.circuittermination",
      "termination_a_id": 134,
      "termination_a": {
        "id": 134,
        "url": "https://netbox.example.com/api/circuits/circuit-terminations/134/",
        "circuit": {
          "id": 85,
          "url": "https://netbox.example.com/api/circuits/circuits/85/",
          "cid": "1"
        },
        "term_side": "Z"
      },
      "termination_b_type": "dcim.interface",
      "termination_b_id": 812,
      "termination_b": {
        "id": 812,
        "url": "https://netbox.example.com/api/dcim/interfaces/812/",
        "device": {
          "id": 35,
          "url": "https://netbox.example.com/api/dcim/devices/35/",
          "name": "imr01-eve-asva02.tingfiber.net",
          "display_name": "imr01-eve-asva02.tingfiber.net"
        },
        "name": "ge-0/0/2",
        "cable": 179,
        "connection_status": {
          "value": true,
          "label": "Connected"
        }
      },
      "type": "",
      "status": {
        "value": "connected",
        "label": "Connected",
        "id": true
      },
      "label": "",
      "color": "",
      "length": null,
      "length_unit": null
    }
  ]
}
```

**Comment by DanSheps** [comment]
It looks like endpoint information is not being built properly

**Comment by DanSheps** [comment]
Duplicate of #4619

**Comment by dgarros** [comment]
We are seeing the exact same issue in 2.8.7 with the fix that associated with #4619. 
As far as I can tell this issue is different than #4619, should I reopen a new issue to track that separately or is it better to reopen this one ?

**Comment by jeremystretch** [comment]
I believe this bug initially was considered a symptom of #4619, however it doesn't seem to have been resolved in the course of that work. #4851 has been raised against v2.8.7, which seems to have recaptured this issue. That bug has been resolved in the upcoming v2.8.8 release.
